demo-site/utils/constants.js

```diff
@@ -0,0 +1,55 @@
+// region imports
+const path = require('path');
+// endregion
+
+const JSON_SPACING = 4;
+
+const SERVER_CONFIG_FILE_NAME = 'px_config.json';
+const SERVER_CONFIG_INC_FILE_NAME = "config.inc.json";
+const PX_ENFORCER_CONFIG_FILE_NAME = 'px_config.json';
+
+const SERVERS_DIRECTORY_NAME = "servers";
+const SERVERS_DIRECTORY_PATH = path.join(__dirname, `../${SERVERS_DIRECTORY_NAME}`);
+
+const FIRST_PARTY_STATIC_FILE_SUFFIX = ".firstparty";
+const THIRD_PARTY_STATIC_FILE_SUFFIX = "";
+
+const CDN_DEPLOY_TOOL_CONFIG_FILE_NAME = "cliConfig.json";
+const CDN_DEPLOY_TOOL_RUN_CLI_COMMAND = "npm run cli";
+const CDN_DEPLOY_TOOL_BUILD_COMMAND = "npm run build";
+
+const PX_METADATA_FILE_NAME = "px_metadata.json";
+
+const TEST_APP_CREDENTIALS_ENDPOINT = "/test-app-credentials";
+const SUPPORTED_FEATURES_ENDPOINT = "/supported-features";
+const CONFIG_ENDPOINT = "/config";
```

Comment:
Re-evaluate if all these constants are necessary? Some of them (especially CDN deploy tool ones) may not be.